Add Two-Person rule (#6)
CopenhagenSoftware/Handbook: README.md

Rushing communication, and especially code reviews, leads to misunderstandings and mistakes.

This will likely end up in more work in the end, and therefore is not a time saver.

> ## We have 4 eyes on production changes

Reviewer: ```suggestion
## We make changes in pairs where regulare review is unavailable
```

> Whenever we change stuff in production we have a collegue looking over our shoulder to validate we do as we intent to do.

Reviewer: ```suggestion
Whenever we change something in production we have a colleague observe to validate we do as we intend to do.
```

> Whenever we change stuff in production we have a collegue looking over our shoulder to validate we do as we intent to do.

Reviewer 2: When performing changes in a production environment, two people must be involved. One doing the change, and one  validating that the changes are implemented as expected. We partner to help eachother catch human errors, that would otherwise slip a person by.

> Whenever we change stuff in production we have a collegue looking over our shoulder to validate we do as we intent to do.

Reviewer 2: ```suggestion
When performing changes in a production environment, two people must be involved. One doing the change, and one validating that the changes are implemented as expected. We partner to help eachother catch human errors, that would otherwise slip a person by.
```

Author: Your comment is now outdated, is it still relevant?

> The term Production covers any live running system upon which manual wrongful changes would cause downtime for the client so also deploy pipelines. 

Reviewer: ```suggestion
The term Production covers any live running system where invalid changes could cause downtime - this includes deployment pipelines. 
```

> The term Changes covers any modification not covered by normal review, normally things not in source control.

Reviewer: ```suggestion
The term Changes covers any modification not covered by normal review. For example, manually modifying configuration values instead of using Infastructure-as-Code kept in source control.
```
